02-wrangling.Rmd

```diff
@@ -316,7 +507,7 @@ a data frame `data`:
 
 One way of doing is to just write multiple lines of code, storing temporary objects as you go:
 
-```
+```{r eval = F}
```

Comment:
Please change all of the `F`'s to `FALSE`. If you don't do this, then you can do crazy things like `F <- TRUE` and then `F` will hold `TRUE` not `FALSE` in those code chunks.
```suggestion
```{r eval = FALSE}
```